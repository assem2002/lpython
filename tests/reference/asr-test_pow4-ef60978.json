{
    "basename": "asr-test_pow4-ef60978",
    "cmd": "lpython --show-asr --no-color {infile} -o {outfile}",
    "infile": "tests/errors/test_pow4.py",
    "infile_hash": "af9b11ec50a82de28afac31faa4a78bdbeb5f317537fc37860b139cd",
    "outfile": null,
    "outfile_hash": null,
    "stdout": null,
    "stdout_hash": null,
    "stderr": "asr-test_pow4-ef60978.stderr",
    "stderr_hash": "ca9308b1cd1d33f159f59c7181ed31e75579eb657f066f97604dee84",
    "returncode": 2
}