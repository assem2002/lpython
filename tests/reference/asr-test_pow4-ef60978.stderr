style suggestion: Could have used '**' instead of 'pow'
  --> tests/errors/test_pow4.py:10:11
   |
10 |     print(pow(x, a, b))
   |           ^^^^^^^^^^^^ '**' could be used instead

semantic error: Arguments do not match for any generic procedure, pow
  --> tests/errors/test_pow4.py:10:11
   |
10 |     print(pow(x, a, b))
   |           ^^^^^^^^^^^^ 


Note: Please report unclear or confusing messages as bugs at
https://github.com/lcompilers/lpython/issues.
